pkg/apis/workflow/v1alpha1/workflow_types.go

```diff
@@ -840,6 +840,8 @@ type PodGC struct {
 	Strategy PodGCStrategy `json:"strategy,omitempty" protobuf:"bytes,1,opt,name=strategy,casttype=PodGCStrategy"`
 	// LabelSelector is the label selector to check if the pods match the labels before being added to the pod GC queue.
 	LabelSelector *metav1.LabelSelector `json:"labelSelector,omitempty" protobuf:"bytes,2,opt,name=labelSelector"`
+	// DelaySeconds is how long to delay pod GC for.
+	DelaySeconds uint32 `json:"delaySeconds,omitempty" protobuf:"bytes,3,opt,name=delaySeconds"`
```

Comment:
Can you change it to `intstr.IntOrString` some it can be parameterized by parameters?

Reply:
@sarabala1979 done, hopefully I got that right.